packages/synthetix-main/cannonfile.toml

```diff
@@ -37,8 +37,8 @@ modified = ["contracts/modules/esnx", "contracts/modules/common", "scripts"]
 [run.deploy_core]
 exec = "scripts/deploy.js"
 func = "deploy"
-args = ["Core", ".*/(core|common)/.*"]
-modified = ["contracts/modules/core", "contracts/modules/common", "contracts/utils", "scripts"]
+args = ["Core", ".*/(core|common|mocks)/.*"]
+modified = ["contracts/modules/core", "contracts/modules/common", "contracts/modules/mocks", "contracts/utils", "scripts"]
```

Comment:
This is interesting. We need to decide how we want to mock modules.

The way I chose to do it is include the mocks in the system, because I found an unfinished implementation for that in the code, and I want to deviate from what you guys are doing as little as possible.

If we do decide this way is ok, I recommend that we see if there is a way to exclude mocks in an official build with cannon.